compiler/rustc_resolve/src/lib.rs

```diff
@@ -1581,6 +1581,13 @@ impl<'a, 'tcx> Resolver<'a, 'tcx> {
         self.get_macro(res).is_some_and(|macro_data| macro_data.ext.builtin_name.is_some())
     }
 
+    fn is_builtin(&mut self, res: Res) -> bool {
```

Comment:
This function is used once and doesn't make much sense outside of that context, better inline and remove.